.eslintrc

```diff
@@ -181,9 +182,9 @@
     "prefer-promise-reject-errors": 0,    // require using Error objects as Promise rejection reasons
     "radix": "error",                     // enforce the consistent use of the radix argument when using parseInt()
     "require-await": 0,                   // disallow async functions which have no await expression
-    "vars-on-top": 0,                     // require var declarations be placed at the top of their containing scope
+    "vars-on-top": "error",               // require var declarations be placed at the top of their containing scope
     "wrap-iife": "error",                 // require parentheses around immediate function invocations
-    "yoda": 0,                            // require or disallow “Yoda” conditions
+    "yoda": "error",                      // require or disallow “Yoda” conditions
```

Comment:
i could relent on this without much argument, but our style checker should already complain if we're doing an assignment in a conditional expression. Besides, how did you manage to mistake `===` for `=`?